spec/javascripts/analytics/static-analytics-spec.js

```diff
@@ -153,6 +153,12 @@ describe("GOVUK.StaticAnalytics", function() {
           number: 34,
           defaultValue: 'other',
           setupArgumentsIndex: 8
+        },
+        {
+          name: 'content-id',
+          number: 4,
+          defaultValue: 'unknown',
```

Comment:
Could this be an empty UUID instead? Just in case we want to later always fit this value into a UUID field in a database. `00000000-0000-0000-0000-000000000000`